opm/input/eclipse/Schedule/SummaryState.hpp

```diff
@@ -58,16 +61,20 @@ class UDQSet;
 //     st.has("WWCT:OPX") => True
 //     st.has_well_var("OPX", "WWCT") => True
 //
-//
 //     // The WGOR:OPY key is added with the general add("WGOR:OPY") and is *not*
 //     // accessible through the specialized st.has_well_var("OPY", "WGOR").
 //     st.has("WGOR:OPY") => True
 //     st.has_well_var("OPY", "WGOR") => False
 
+namespace Opm {
```

Comment:
That the documentation of SummaryState is outside of the namespace OPM but SummaryState is declared in it,  seems a bit strange. On the other hand it it not a doxygen documentation.